server/commands.py

```diff
@@ -425,6 +425,20 @@ def ooc_cmd_unban(client, arg):
     except ServerError:
         raise
     logger.log_server('Unbanned {}.'.format(ip), client)
+    client.send_host_message('Unbanned {}'.format(ip))
+
+def ooc_cmd_unbanhdid(client, arg):
+    if not client.is_mod:
+        raise ClientError('You must be authorized to do that.')
+    hdid = arg.strip()
+    if len(hdid) < 8:
```

Comment:
If you prevent short HDIDs from being parsed, you are essentially disallowing yourself from banning potentially abusive and invalid HDIDs.